src/dodal/devices/oav/oav_to_redis_forwarder.py

```diff
@@ -86,7 +86,7 @@ async def _get_frame_and_put_to_redis(self, response: ClientResponse):
         await self.redis_client.expire(sample_id, timedelta(days=self.DATA_EXPIRY_DAYS))
         LOGGER.debug(f"Sent frame to redis key {sample_id} with uuid {image_uuid}")
 
-    async def _open_connection_and_(self, function_to_do: Callable):
+    async def _open_connection_and_do_function(self, function_to_do: Callable):
```

Comment:
Could: Sorry I should've said this before, but it might be nice to type hint `function_to_do` so that it must accept `ClientResponse` as a parameter, and an optional `str`